grades/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.conf import settings
+# Each user can add, modify or remove course from is list
+# Each course contains coures id(default by Django), name, credinitials and grade
+
+
+class Course (models.Model): 
+    course_name = models.CharField(max_length=50)
+    credinitials = models.IntegerField(max_length=1)
+
+class Grade(models.Model):
+    SEMESTER_NAME = (
+        ('A', 'Winter'),
+        ('B', 'Spring'),
+        ('C', 'Summer'),
+    )
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,on_delete=models.CASCADE,related_name='%(class)s_student')
+    course = models.ForeignKey(Course, on_delete=models.RESTRICT)
+    grade = models.IntegerField(max_length=3)
+    semester = models.CharField(max_length=1, choices=SEMESTER_NAME)
+    year = models.IntegerField(max_length=4)
+
+    @classmethod
+    def addcourse(cls, current_user, new_course):
+        new_course.user.add(current_user)
```

Comment:
If 'user' field is a foreign key then it's not a list, it's pointing to a specific one user.
if that is the case then it doesn't have a method 'add'